cxx4opencl/address_spaces.txt

```diff
@@ -259,26 +353,28 @@ during type deduction if it is not explicitly provided in the
 instantiation.
 
 [source,cpp]
-----------
-template<typename T>
-void foo(T* i){
-  T var;
- }
-
- __global int g;
-void bar(){
-  foo(&g); // error: template instantiation failed as function scope variable 'var'
-           // appears to be declared in __global address space (see line 3).
-}
-----------
+------------
+1  template<typename T>
+2  void foo(T* i){
+3   T var;
+4  }
+5 
+6  __global int g;
+7  void bar(){
+8    foo(&g); // error: template instantiation failed as function scope variable 'var'
+9             // appears to be declared in __global address space (see line 3).
+10 }
```

Comment:
Unless they are explicitly referred to, I'd remove the line numbers.

Reply:
we do in the comment on line 9
>// appears to be declared in __global address space (see line 3).